KIF Tests/AccessibilityIdentifierTests.m

```diff
@@ -47,12 +47,14 @@ - (void)testWaitingForAbscenceOfViewWithAccessibilityIdentifier
 
 - (void)testLongPressingViewWithAccessibilityIdentifier
 {
+  [tester tapViewWithAccessibilityIdentifier:@"idGreeting"];
```

Comment:
I wonder if it makes sense to add a `showMenu` text action (possibly with a version check if backwards compatibility is an issue) that executes the tap+long press.

Reply:
Would you add an issue for an improvement to add that and we will get to that when we can.